apps/ios/Shared/Model/SuspendChat.swift

```diff
@@ -134,20 +136,33 @@ func initChatAndMigrate(refreshInvitations: Bool = true) {
     }
 }
 
-func startChatAndActivate(dispatchQueue: DispatchQueue = DispatchQueue.main, _ completion: @escaping () -> Void) {
+func startChatForCall() {
```

Comment:
add a comment that the main difference with startChatAndActivate is that it doesn't wait for NSE? If I understand correctly. A bit hard to see behind different calls to activateChat(), via activate() in startChatAndActivate which does the same but with completion()